test/logflare/backends/adaptor/postgres_adaptor/pipeline_test.exs

```diff
@@ -55,18 +57,29 @@ defmodule Logflare.Backends.Adaptor.PostgresAdaptor.PipelineTest do
     test "ingests dispatched message", %{
       memory_buffer_pid: memory_buffer_pid,
       repository_module: repository_module,
-      source_backend: %{source: source}
+      source_backend: source_backend
     } do
       log_event =
         build(:log_event,
           token: TestUtils.random_string(),
-          source: source,
+          source: source_backend.source,
           body: %{"data" => "data"}
         )
 
       MemoryBuffer.add(memory_buffer_pid, log_event)
 
-      fetcher = fn -> repository_module.all(LogEvent) end
+      fetcher = fn ->
+        repository_module.all(
+          from(l in Repo.table_name(source_backend),
+            select: %LogEvent{
+              id: l.id,
+              body: l.body,
```

Comment:
as above, not quite sure what this field refers to.

Reply:
https://github.com/Logflare/logflare/pull/1580#discussion_r1242588817